symphony/lib/toolkit/class.general.php

```diff
@@ -1388,11 +1388,14 @@ public static function countWords($string)
      *  true if the ellipses should be appended to the result in circumstances
      *  where the result is shorter than the input string. false otherwise. this
      *  defaults to false.
+     * @param string $character (optional)
```

Comment:
Is there any way we can had a `@since Symphony 2.7` only for a param? @nitriques

Reply:
Yes we do, but I would try to avoid adding a parameter.